### DISCUSSION TITLE
It's a question rather than an issue

### DISCUSSION THREAD

**Discussion by florianionascu7** [NVDARemote/NVDARemote#245] [Ideas]
Hello! As I translated the interface file for this add-on into Romanian, I wanted to submit it to developers. So, I've installed Tortoise Git and Git for windows, I cloned this repository, added the „ro” folder in the „locale” folder, clicked on the „pull” option and pressed the OK button. Did I do something wrong? Did I damage something? Can someone explain me the correct way in which I can make commits and pulls using Tortoise Git? I have never worked with it up to now, only with Tortoise SVN.

**Comment by angeloabrantes** [comment]
I've the repo in my tortoise, but the option comit doesn't apear in my menu.

Thanks.

Ângelo Abrantes


Às 19:54 de 01-10-2018, florianionascu7 escreveu:
>
> Hello! As I translated the interface file for this add-on into 
> Romanian, I wanted to submit it to developers. So, I've installed 
> Tortoise Git and Git for windows, I cloned this repository, added the 
> „ro” folder in the „locale” folder, clicked on the „pull” option and 
> pressed the OK button. Did I do something wrong? Did I damage 
> something? Can someone explain me the correct way in which I can make 
> commits and pulls using Tortoise Git? I have never worked with it up 
> to now, only with Tortoise SVN.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/NVDARemote/NVDARemote/issues/196>, or mute the 
> thread 
> <https://github.com/notifications/unsubscribe-auth/AfcMoZmXkNuc-luwdWyyV5wRP2Ik2LWnks5ugmTrgaJpZM4XCrfO>.
>



---
Este e-mail foi verificado em termos de vírus pelo software antivírus Avast.
https://www.avast.com/antivirus

**Comment by florianionascu7** [comment]
I am back with a problem. I gave Tortoise Git up and I did a SVN checkout to this repository. I wanted to commit, but after I entered my Github username and password and pressed the OK button, a commit fail error occured, though the username and the password were inserted correctly. What can I do?

**Comment by LeonarddeR** [comment]
SVN is most definitely going to work here. I'm afraid you will receive great help if you ask your question on the NVDA translations mailing list.

**Comment by alekssamos** [comment]
1. Press the button "Fork your own copy of NVDARemote/NVDARemote to your account";
2. Go to command line;
3. Go to directory `cd /d E:\remote\addon`;
4. `git add .`
5. `git commit -m "message";`
6. `git pull`
7. Create a pull request
there were no problems.

**Comment by bdorer** [comment]
As the question has answered, please close the issoe.
